src/domains/Task/components/TaskAdvancedView/TaskAdvencedView.js

```diff
@@ -1,11 +1,28 @@
 import { TaskList, TaskSimpleForm } from '~/domains/Task'
-import { Box, Center, Flex, Space } from '@mantine/core'
+import { Box, Flex, Space } from '@mantine/core'
+import * as TASK_CONTEXT_ACTIONS from '~/domains/Task/context/__constants__/taskActions'
+import { useTaskDispatchContext } from '~/domains/Task/context'
+
 const TaskAdvancedView = () => {
+  const taskDispatch = useTaskDispatchContext()
+
+  const handleSubmitTask = (props) => {
```

Comment:
I am strongly recommended you to create custom hoot useTaskActions and move it to hook